webviz_config/themes/default_assets/default_logo.svg

```diff
@@ -1,281 +1,51 @@
+<path d="m-281.29 143.17q-4.8068 0-7.5417-2.4034-2.652-2.4863-2.652-7.3759 0-1.989 0.41438-3.978l5.967-28.178h11.934l-5.967 28.178q-0.41437 2.0719-0.41437 2.9835 0 2.7349 2.8178 2.7349 3.8123 0 7.0444-4.8068 3.315-4.8068 5.304-11.437t1.989-11.437q-0.16575 0.41438-1.0774 0.66301-0.91163 0.16575-1.5746 0.16575-1.4089 0-2.2376-1.326-0.74587-1.326-0.74587-2.9006 0-1.989 1.2431-3.2321 1.326-1.2431 3.8951-1.2431 2.7349 0 3.978 2.0719 1.2431 1.989 1.2431 5.1383 0 7.4588-2.5691 15.995-2.5691 8.4533-7.8732 14.42t-13.177 5.967z"/>
+<path d="m-242.58 96.763q-2.7349 0-4.641-1.9061t-1.9061-4.641 1.9061-4.641q1.9061-1.989 4.641-1.989t4.641 1.989q1.989 1.9061 1.989 4.641t-1.989 4.641q-1.9061 1.9061-4.641 1.9061zm-5.6355 46.41q-3.978 0-6.4643-2.4863-2.4034-2.4863-2.4034-7.4588 0-2.0719 0.663-5.3869l5.6355-26.603h11.934l-5.967 28.178q-0.3315 1.2431-0.3315 2.652 0 1.6575 0.74588 2.4034 0.82875 0.66301 2.652 0.66301 1.4918 0 2.652-0.49726-0.3315 4.1438-2.9835 6.3814-2.5691 2.1548-6.1328 2.1548zm30.25-37.46h-4.641q-2.3205 0-3.4808 0.24863-1.0774 0.24862-1.0774 1.0774 0 0.16575 0.41438 0.41437 0.49725 0.16575 0.49725 1.4089 0 2.1548-1.326 3.315-1.326 1.0774-3.1493 1.0774-1.5746 0-2.7349-0.91163-1.0774-0.9945-1.0774-2.7349t1.1602-3.7294q1.2431-1.989 3.3979-3.315 2.2376-1.326 5.0554-1.326h21.299l-20.304 35.056q0.49725 0 1.8233 0.16575 2.8178 0.3315 4.4753 0.3315 3.5636 0 3.8951-1.6575-0.74587-0.0829-1.1602-0.49726-0.41438-0.49725-0.41438-1.2431 0-1.326 1.2431-2.4034 1.2431-1.1603 3.3979-1.1603 1.8233 0 2.7349 1.0774t0.91163 2.9006q0 1.989-1.2431 4.0609t-3.5636 3.4808q-2.3205 1.326-5.304 1.326h-21.962z"/>
+<path d="m-248.2-2.41c4.308-3.2104 6.7632-7.0502 8.5586-9.8935 1.3137 2.2444 10.384 20.083 10.296 22.472-0.70903 0.7402-8.5453 6.2423-13.289 9.2298-0.33409-7.6902-2.2555-12.855-5.5657-21.808z" fill="url(#j)"/>
+<path d="m-271.9-42.718c0.97416-1.5374 3.6124-6.3645 4.324-7.905 13.36 13.093 11.314 11.838 16.927 21.326-0.69307 1.2215-4.4606 7.5904-5.3824 9.0668-1.0723-2.1445-9.9738-14.918-15.868-22.488z" fill="url(#k)"/>
+<path d="m-261.96-63.625-3.2128 9.0715 20.978 12.284 5.4806-12.662z" fill="url(#i)"/>
+</g>
+<g fill="#808080" stroke-width=".10361" aria-label="TM">
+<path d="m60.806 1.9142v2.5356h-0.38448v-2.5356h-0.97942v-0.31568h2.3433v0.31568z"/>
+<path d="m64.646 4.4498v-1.9022q0-0.31568 0.01821-0.60708-0.09916 0.36222-0.17808 0.56661l-0.73659 1.9427h-0.27116l-0.74671-1.9427-0.11332-0.34401-0.06678-0.2226 0.0061 0.22462 0.0081 0.38246v1.9022h-0.34404v-2.8513h0.50792l0.75885 1.9771q0.04047 0.11939 0.0769 0.257 0.03845 0.13558 0.05059 0.19629 0.01619-0.080944 0.06678-0.24486 0.05261-0.16594 0.07083-0.20843l0.74469-1.9771h0.49578v2.8513z"/>
+</g>
```

Comment:
Maybe it is possible for us to remove this file, and instead here https://github.com/equinor/webviz-config/blob/96d077306ba701dfac4ca1c78a95e43cbef54eac/webviz_config/themes/_default_theme.py#L10-L12 we point to the logo already in https://github.com/equinor/webviz-config/blob/master/webviz_config/_docs/static/webviz-logo.svg? Then we only have the logo defined one place.